xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/uicomponents/extension/extension.css

```diff
@@ -475,3 +475,83 @@ ul.collapsible.document-tree {
   font-weight: normal;
   padding-right: .3em;
 }
+
+@media screen and (min-width: 675px) {
```

Comment:
Media Queries values are very random. We use the breakpoints from Bootstrap documentation (768px, 992px, 1200px), see https://getbootstrap.com/docs/3.4/css/#grid-media-queries